app/main.py

```diff
@@ -1 +1,8 @@
 # write your code here
+def copy_file(command: str) -> None:
+    if len(command.split()) == 3:
+        old_file = command.split()[1]
+        new_file = command.split()[2]
```

Comment:
```suggestion
        command, old_file, new_file = command.split()
```
No need to split three times